fiorella/src/styles/pages/_product.scss

```diff
@@ -0,0 +1,127 @@
+.title {
+    font-family: $merriweathersans;
+    font-size: 28px;
+    margin: 0;
+    padding: 0;
+    text-align: center;
+
+    @media #{ $small-desktop-view } {
+      display: none;
+    }
+}
+
+.super-product {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  height: auto;
+  justify-content: center;
+  width: 100%;
+
+  @media #{ $small-desktop-view } {
+    flex-direction: row;
+    height: 100%;
+    justify-content: space-between;
+    width: 50%;
+  }
+
+  .box-product {
+    display: flex;
+    height: 400px;
+    justify-content: center;
+    width: 100%;
+
+    .product {
+      height: 100%;
+      justify-content: center;
+      object-fit: contain;
+      width: 100%;
+    }
+
+    @media #{ $small-desktop-view } {
+      width: 100%;
+    }
+  }
+
+  .box-micro {
+    display: none;
+    justify-content: center;
+    padding-left: 20px;
+    padding-top: 25px;
+    width: 70%;
+
+    @media #{ $small-desktop-view } {
+      display: flex;
+    }
+
+    a {
+      height: 85px;
+      margin-right: 20px;
+      padding-top: 10px;
+      width: 95%;
+
+      .thumbnail-pink {
+        height: 75px;
+        width: 75px;
+
+        &:hover {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 4